qiskit/circuit/library/n_local/n_local.py

```diff
@@ -306,6 +347,80 @@ class NLocal(BlueprintCircuit):
 
         The :func:`.n_local` function constructs a functionally equivalent circuit, but faster.
 
+    Examples::
```

Comment:
It seems your old changes re-appeared, please make sure you apply the changes we discussed here too